var/spack/repos/builtin/packages/coreutils/package.py

```diff
@@ -22,3 +25,14 @@ class Coreutils(AutotoolsPackage, GNUMirrorPackage):
     version('8.23', sha256='ec43ca5bcfc62242accb46b7f121f6b684ee21ecd7d075059bf650ff9e37b82d')
 
     build_directory = 'spack-build'
+
+    def configure_args(self):
+        spec = self.spec
+        configure_args = []
+        if spec.satisfies('platform=darwin'):
+            configure_args.append('--program-prefix=g')
```

Comment:
I _think_ I'm okay with this decision, but let's wait and here from other users.

Reply:
Ok I'll bite.  I use this without the prefix all the time.  I think this should be an option, like `+prefix`.  Homebrew ships coreutils with `libexec/gnubin` containing the non-prefixed binaries, so you can actually add it to your path.

Was there an issue with using this without the prefix?  I've been using it that way on my mac for a while.